frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -587,6 +587,7 @@ function VirtualKeyPopup:init()
             }
         },
     }
+    self.tap_interval_override = G_reader_settings:readSetting("ges_tap_interval_on_keyboard") or 0
```

Comment:
Would it make sense to 'nil' it instead of setting it to 0 in the fallback?

(Again, that may not be a great idea, and might make the logic/intent less clear).

Reply:
`nil` 'ing it would make the keyboard not override and use the generic `Tap interval`.
I think it would make the 2 menu items a bit unintuitive:
```
Tap interval
Tap interval on keyboard
```
Currently, the first can be seen as "everywhere else but with the keyboard).

With `nil`, it would mean "everywhere else including the keyboard, until you once use and apply `Tap interval on keyboard` " - which would set it to something else than nil.
Or I would have to save nil when the spinwidget value is 0 - making that number picker unintuitive - and this would prevent getting 0 with the keyboard and 400ms for the generic value (my use case :).